### ISSUE TITLE
Use stimulus-timeago instead of knplabs time bundle

### ISSUE THREAD

**Issue by tacman** [Spomky-Labs/phpwa-demo#7]
The problem with 

```twig
{{ item.createdAt|time_diff }}
```

is that it renders HTML with the age of the item _at the time of rendering_.

What we want is the age of the item.  This can be done by installing stimulus-timeago, and instead using 

```twig
<time data-controller="timeago"
      data-timeago-add-suffix-value="true"
      data-timeago-include-seconds-value="true"
      data-timeago-refresh-interval-value="1000"
      data-timeago-datetime-value="{{ item.createdAt|date('c') }}"></time>.
```

That way, when it runs offline the time ago is correct.  And even updated dynamically!

Add an item at 

https://phpwa-demo.survos.com/

Go offline and refresh, the item age is accurate.

**Comment by Spomky** [comment]
Indeed. It looks better and will perfectly fit in case data are loaded async

**Comment by Spomky** [comment]
It is not needed anymore so I am closing this issue.
